pyproject.toml

```diff
@@ -118,6 +118,9 @@ lightningpose = [
 medpc = [
     "ndx-events==0.2.1",
 ]
+facemap = [
+    "ndx-facemap-motionsvd @ git+https://github.com/catalystneuro/ndx-facemap-motionsvd.git",
```

Comment:
Note that we can't release to pip with repo based dependencies. Do you know if there is a timeline for release?